.github/workflows/check-providers.yml

```diff
@@ -0,0 +1,40 @@
+name: Check Providers
+
+on:
+  pull_request:
+    branches:
+      - main
+  schedule:
+    - cron: "0 12 * * *"
+
+jobs:
+  check-providers:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Clone @api3/chains
+        uses: actions/checkout@v3
+
+      - name: Setup Node
+        uses: actions/setup-node@v3
+        with:
+          node-version: '18'
+          cache: 'yarn'
+
+      - name: Get yarn cache directory path
+        id: deps-cache-dir-path
+        run: echo "dir=$(yarn cache dir)" >> $GITHUB_OUTPUT
+
+      - uses: actions/cache@v3
+        id: deps-cache
+        with:
+          path: ${{ steps.deps-cache-dir-path.outputs.dir }}
+          key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-yarn-
+
+      - name: Install Dependencies
+        run: yarn install --frozen-lockfile
+
+      - name: Run providers:ping
```

Comment:
We'll need to decide on a channel to push these error messages to

Reply:
Could unarchive "technical-notifications"? https://app.slack.com/client/T0104044FT5/C01R5LMRGV7

Reply:
Yeah that could work